processor/configuration/src/main/resources/schemas/actions/google_pubsub_sink_0.1.json

```diff
@@ -19,10 +19,20 @@
     },
     "gcp_service_account_key": {
       "title": "Service Account Key",
-      "description": "The Service account key that can be used as credentials for the PubSub publisher/subscriber in base64 encoding.",
-      "type": "string",
       "x-group": "credentials",
-      "format": "base64"
+      "oneOf": [
+        {
+          "title": "Service Account Key",
+          "description": "The service account key to use as credentials for the Pub/Sub publisher/subscriber. You must encode this value in base64.",
+          "type": "string",
+          "format": "password"
+        },
+        {
+          "description": "An opaque reference to the aws_access_key",
+          "type": "object",
+          "additionalProperties": false
```

Comment:
I guess this is wrong? It should look like [this](https://github.com/bf2fc6cc711aee1a0c2a/cos-fleet-catalog-camel/blob/main/etc/kubernetes/manifests/base/connectors/connector-catalog-camel-social/slack_sink_0.1.json#L64). 
Same thing for `gcp_service_account_key` inside `google_pubsub_source_0.1.json`

Reply:
I guess so, good spot. I'd say we move forward with this PR and raise the issue to the MC team